.travis.yml

```diff
@@ -15,17 +26,27 @@ cache:
   directories:
     - $HOME/.cache/pip
 
-python:
-  - "2.7"
-  - "3.3"
-  - "3.4"
-  - "3.5"
-
 env:
   global:
     - TRIAL_REPORTER=text
 
+# Travis-CI has an old version of PyPy so we install a newer version on the
+# fly.
+# Source: https://github.com/saghul/pycares/blob/master/.travis.yml
 install:
+  - |
+      if [ "$TRAVIS_PYTHON_VERSION" = "pypy" ]; then
+        export PYENV_ROOT="$HOME/.pyenv"
+        if [ -f "$PYENV_ROOT/bin/pyenv" ]; then
+          pushd "$PYENV_ROOT" && git pull && popd
+        else
+          rm -rf "$PYENV_ROOT" && git clone --depth 1 https://github.com/yyuu/pyenv.git "$PYENV_ROOT"
+        fi
+        export PYPY_VERSION="5.3"
+        "$PYENV_ROOT/bin/pyenv" install --skip-existing "pypy-$PYPY_VERSION"
+        virtualenv --python="$PYENV_ROOT/versions/pypy-$PYPY_VERSION/bin/python" "$HOME/virtualenvs/pypy-$PYPY_VERSION"
+        source "$HOME/virtualenvs/pypy-$PYPY_VERSION/bin/activate"
```

Comment:
Thanks. I will give it a try... but first I will use pypy 5.1 as coverage is not documented to work with pypy 5.3 http://coverage.readthedocs.io/en/latest/

again... I have never used pypy so I have no idea about all these crazy version numbers in pypy and pypy3 :)